app/settings/surveys/survey-editor.directive.js

```diff
@@ -122,7 +122,8 @@ function SurveyEditorController(
                         priority: 0,
                         required: false,
                         type: 'post',
-                        show_when_published: 1,
+                        show_when_published: false,
```

Comment:
Note: this default is different than what we have in develop . Develop has `true` instead for show_when_published.